README.md

```diff
@@ -229,15 +221,19 @@ create a symlink called `python2.exe` via `mklink "C:\Python27\python2.exe" "C:\
 and add the directory to your path.
```

Comment:
It looks like this one was out of place from the beginning, https://github.com/MadLittleMods/node-usb-detection/commit/907cfae88f1e3216ad69220da029e43e0a1154a1

Idk, bad edit or rebase